src/LaborMarket/LaborMarket.sol

```diff
@@ -74,89 +74,20 @@ contract LaborMarket is
 
     modifier onlyMaintainer() {
         require(
-            participationBadge.balanceOf(
-                msg.sender,
-                configuration.participationTokenId
-            ) >=
-                (network.baseMaintainerThreshold() *
-                    configuration.repMaintainerMultiplier),
+            reputationModule.getAvailableReputation(
+                address(this),
+                msg.sender
+            ) >= reputationModule.getMaintainerThreshold(address(this)),
             "LaborMarket::onlyMaintainer: Not a maintainer"
         );
         _;
     }
 
-    /// @notice emitted when a new labor market is created
```

Comment:
These events should not have been removed as events are not used in interfaces. Please revert.